### DISCUSSION TITLE
What's the best way to trigger a build when reloading?

### DISCUSSION THREAD

**Discussion by zalo** [yandeu/five-server#100] [Help / Questions]
I have a site that rebuilds pretty quickly with ESBuild (using `npm run build`), what's the best way to get Five Server to run this before serving the page?   @yandeu

**Comment by yandeu** [comment]
Are you using the npm package or the vscode extension?

**Reply by zalo** [reply]
I’m using the VS Code Extension

**Reply by yandeu** [reply]
You mean clicking on "Go Live" should trigger a build (and watch) using your npm build script?
I don't think you can execute a script using Five Server (vscode).
But if you use five-server from npm, you could integrate it inside your npm build script.

**Reply by zalo** [reply]
Ideally every time the watch refreshes, I could get it to call a command (with a cooldown so it doesn’t create a loop) :-)
